src/library_egl.js

```diff
@@ -645,12 +643,12 @@ var LibraryEGL = {
   eglGetCurrentSurface: function(readdraw) {
     if (readdraw == 0x305A /* EGL_READ */) {
       return EGL.currentReadSurface;
-    } else if (readdraw == 0x3059 /* EGL_DRAW */) {
+    }
+    if (readdraw == 0x3059 /* EGL_DRAW */) {
```

Comment:
I find else-if actually easier to read for a thing like this, where it's a series of ifs on the same thing. Do you disagree?

Reply:
Maybe marginally easier to read in some cases.  But I think that benefits outweigh the costs:

1. less code
2. less indentation
3. Single rule for all cases ("else after return is always wrong") keeps things simple.

Since tidying up this stuff in various places I've trained my self to see else after return a code smell, so for me the old code now looks wrong :)